engine/gameobject/src/dmsdk/gameobject/res_lua.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 The Defold Foundation
+// Licensed under the Defold License version 1.0 (the "License"); you may not use
+// this file except in compliance with the License.
+//
+// You may obtain a copy of the License, together with FAQs at
+// https://www.defold.com/license
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+#ifndef DMSDK_GAMEOBJECT_RES_LUA_H
+#define DMSDK_GAMEOBJECT_RES_LUA_H
+
+#include <stdint.h>
+
+#include <ddf/ddf.h>
+#include <gameobject/lua_ddf.h>
```

Comment:
The "_ddf.h" aren't under the actual "dmsdk" folder, in part to signal that they are more "volatile" than the other headers.